app/MMB/src/scenes/tripServices/insurance/insuranceSelectionScene/IconWithText.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation } from '@kiwicom/mobile-localization';
+import { Text, TextIcon, Color } from '@kiwicom/mobile-shared';
+
+type IconWithTextProps = {
+  +textIconCode: string,
+  +text: React.Element<typeof Translation>,
+  +color: string,
+};
+
+export default class IconWithText extends React.Component<IconWithTextProps> {
+  static defaultProps = {
+    color: Color.ink.light,
+  };
+  render() {
+    return (
+      <View
+        style={{
+          flexDirection: 'row',
+          alignItems: 'center',
+          marginTop: 2,
+          marginBottom: 2,
+        }}
```

Comment:
Please don't use inline styles where it is not needed. These styles will be re-created on every render.

Reply:
I had realised this when I asked you to review, fixed now.